api/services/auth.go

```diff
@@ -33,6 +34,11 @@ type AuthService interface {
 }
 
 func (s *service) AuthDevice(ctx context.Context, req *models.DeviceAuthRequest, remoteAdrr string) (*models.DeviceAuthResponse, error) {
+	duration, err := rand.Int(rand.Reader, big.NewInt(5))
+	if err != nil {
+		return nil, err
+	}
+	time.Sleep(time.Second * time.Duration(duration.Int64()))
```

Comment:
Why? I don't see a reason to sleep here. Please provide a background reasining.

Reply:
This is a try to spread the auth ping's peak. I know, it seems to be a little dummy, but I've said, it is just a test.

I have forgotten to set this PR as a Draft.